doc/source/cluster/running-applications/job-submission/quickstart.rst

```diff
@@ -111,12 +111,19 @@ Make sure to specify the path to the working directory in the ``--working-dir``
     # Job 'raysubmit_inB2ViQuE29aZRJ5' succeeded
     # ------------------------------------------
 
-This command will run the script on the Ray Cluster and wait until the job has finished. Note that it also streams the stdout of the job back to the client (``hello world`` in this case). Ray will also make the contents of the directory passed as `--working-dir` available to the Ray job by downloading the directory to all nodes in your cluster.
+This command will run the entrypoint script on the Ray Cluster's head node and wait until the job has finished. Note that it also streams the stdout of the job back to the client (``hello world`` in this case). Ray will also make the contents of the directory passed as `--working-dir` available to the Ray job by downloading the directory to all nodes in your cluster.
```

Comment:
It not only streams STDOUT but also STDERR. You can do a simple experiment:

```python
# example.py
import sys

# Print a message to stdout
print("This is a message to stdout")

# Print a message to stderr
print("This is an error message to stderr", file=sys.stderr)
```
<img width="703" alt="Screen Shot 2023-11-21 at 3 39 49 PM" src="https://github.com/ray-project/ray/assets/20109646/3202c424-a4b2-4267-8184-54cacabadb45">

Reply:
Good point. I just want to convey that it would stream whatever is normally output to the terminal when you run a command in your local terminal.  Should I say "streams the stdout and stderr" or  "streams the output"?

Reply:
Maybe the former? A lot of commands/tools do not stream stderr by default.

Reply:
Updated to "streams the output of the entrypoint script", which should be clear. 61d65ff

Reply:
Oh sorry, didn't see your message. Updated to  "streams the stdout and stderr" ab7950c35c